.env.sample

```diff
@@ -8,11 +8,14 @@ REPAIRS_SERVICE_API_KEY=repairsApiKey
 GSSO_URL=https://auth.hackney.gov.uk/auth?redirect_uri=
 GSSO_TOKEN_NAME=hackneyToken
 HACKNEY_JWT_SECRET=sekret
-HACKNEY_AUTHORISED_GROUP=repairs-hub-frontend-staging
+REPAIRS_AGENTS_GOOGLE_GROUPNAME=repairs-hub-frontend-staging
+CONTRACTORS_GOOGLE_GROUPNAME=repairs-hub-frontend-staging-contractors-alphatrack
 
 #Redirect URL to run code locally
 REDIRECT_URL=localhost:5000
 
 #E2E
 CYPRESS_GSSO_TEST_KEY=
```

Comment:
How about deleting this since is not used anymore?

Reply:
Thanks. Will do this separately.